### PR TITLE
Link GitHub api

### PR DESCRIPTION
**Pull request by borek2**
I have linked all the pullrequests that are requested by graphs to its apis (Github AND Bitbucket). With this we can check whether pull requests are merged or not (or some more info depending on the transformer configuration). When a pull request cannot be found, it returns an empty object (because of test data this is possible). Also it will probably log some errors, but you can ignore them, this is jquery logging everything

### CODE REVIEW

File: src/aggregators/graph4aggregator.js [hunk 1]
```diff
@@ -34,16 +35,17 @@ function Graph4Aggregator(owner, reponame, pullRequestNumber) {
     }
     
     function createMatrix(pullRequest) {
+        console.log(pullRequest);
```
Comment (lvdoorn) [inline src/aggregators/graph4aggregator.js]: Maybe we should not have this in the master branch.

Comment (borek2) [inline src/aggregators/graph4aggregator.js]: yes sorry :D

File: src/example-services.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-/*globals Graph4Aggregator, console*/
+/*globals Graph2Aggregator, console*/
 
-new Graph4Aggregator("mboom", "TI2806", 1).then(function (result) {
+new Graph2Aggregator("mboom", 5).then(function (result) {
     console.log(result);
```
Comment (mboom) [inline src/example-services.js]: Do we need the console log here?

File: src/example-services.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-/*globals Graph4Aggregator, console*/
+/*globals Graph2Aggregator, console*/
 
-new Graph4Aggregator("mboom", "TI2806", 1).then(function (result) {
+new Graph2Aggregator("mboom", 5).then(function (result) {
```
Comment (mboom) [inline src/example-services.js]: Username and repository name are hardcoded. Maybe we should use global vars?

Comment (lvdoorn) [inline src/example-services.js]: I don't think we should put too much attention on the example-services file.

Comment (borek2) [inline src/example-services.js]: As this is purely for testing purposes, these will be removed, so no bother to make them global

**Comment by mboom** [comment]
@lvdoorn The point is, it will Always push text to the browser console. That could be confusing while debugging.

**Comment by borek2** [comment]
Ill remove it

**Pull request merged**